server/src/test/java/org/elasticsearch/index/mapper/NumberFieldMapperTests.java

```diff
@@ -379,6 +383,7 @@ protected final class NumberSyntheticSourceSupport implements SyntheticSourceSup
         private final Function<Number, Number> round;
         private final Long nullValue = usually() ? null : randomNumber().longValue();
         private final boolean coerce = rarely();
+        private final boolean ignoreMalformed = rarely();
```

Comment:
Do we have a specific test for this somewhere?

Reply:
It's run `rarely` like null value support or coerce or whatever. I could could run it on each run in a new test if you'd like. That way we're more likely to catch regressions faster....

Reply:
Yeah, I think I'd be happier with explicit tests for each configuration.

Reply:
I've opened #90565 so we'll have known examples to test in addition to some tests each iteration.